tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -86,12 +63,24 @@ echo "ADMIN_ACCESS=${ADMIN_ACCESS}"
 # cilium does not yet pass conformance tests (shared hostport test)
 #create_args="--networking cilium"
 create_args=()
-create_args=("--network-cidr=10.0.0.0/16,10.1.0.0/16,10.2.0.0/16,10.3.0.0/16,10.4.0.0/16,10.5.0.0/16,10.6.0.0/16,10.7.0.0/16,10.8.0.0/16,10.9.0.0/16,10.10.0.0/16,10.11.0.0/16,10.12.0.0/16")
-create_args+=("--networking=${CNI_PLUGIN:-calico}")
+if [[ "${CLOUD_PROVIDER}" == "aws" ]]; then
+  create_args+=("--network-cidr=10.0.0.0/16,10.1.0.0/16,10.2.0.0/16,10.3.0.0/16,10.4.0.0/16,10.5.0.0/16,10.6.0.0/16,10.7.0.0/16,10.8.0.0/16,10.9.0.0/16,10.10.0.0/16,10.11.0.0/16,10.12.0.0/16")
+  create_args+=("--node-size=t3a.medium,t3.medium,t2.medium,t3a.large,c5a.large,t3.large,c5.large,m5a.large,m6a.large,m5.large,c4.large,c7a.large,r5a.large,r6a.large,m7a.large")
+  create_args+=("--node-volume-size=20")
+  create_args+=("--zones=us-east-2a,us-east-2b,us-east-2c")
+  create_args+=("--image=${INSTANCE_IMAGE:-ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id}")
+fi
+if [[ "${CLOUD_PROVIDER}" == "gce" ]]; then
+  create_args+=("--zones=us-east1-b,us-east1-c,us-east1-d")
+  create_args+=("--topology=private")
```

Comment:
If we want to use private topology, we need to either:
- create a new topology called `private-nodes` which creates nodes with internal IPs only and control planes with both external and internal IPs
- create the cluster with `--bastion=e2e-bastion` flag, set the `KUBE_SSH_BASTION` env variable and tweak cl2 to use that to connect to the control plane instances. the e2e framework supports this https://github.com/kubernetes/kubernetes/blob/735804dc812ce647f8c130dced45b5ba4079b76e/test/e2e/framework/ssh/ssh.go#L53